content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -173,28 +170,34 @@ python ${SCRIPTS}create_hashed_signed_statement.py \
 1. Submit the Signed Statement to DataTrails, using the credentials in the `DATATRAILS_CLIENT_ID` and `DATATRAILS_CLIENT_SECRET`.
 
     ```bash
-    python ${SCRIPTS}register_signed_statement.py \
-      --signed-statement-file $SIGNED_STATEMENT_FILE \
+    python -m datatrails_scitt_samples.scripts.register_signed_statement \
+      --signed-statement-file signed-statement.cbor \
       --output-file $TRANSPARENT_STATEMENT_FILE \
       --log-level INFO
     ```
 
+    Find and copy the leaf hash from the output. It will look like this:
+    ```
+    INFO:register-statement:Leaf Hash: 30f5650fbe3355ca892094a3fbe88e5fa3a9ae47fe3d0bbace348181eb2b76db
+    ```
+
 1. View the Transparent Statement, as a result of registering the Signed Statement
 
     ```bash
-    python datatrails_scitt_samples/dump_cbor.py \
-      --input $TRANSPARENT_STATEMENT_FILE
+    python -m datatrails_scitt_samples.dump_cbor \
+      --input transparent-statement.cbor
     ```
 
-<!-- 
-TODO: Update with MMR verification
-1. Verify the signature of the receipt
+1. Verify the the receipt
 
     ```bash
-    python ${SCRIPTS}/verify_receipt_signature.py \
-      --transparent-statement-file $TRANSPARENT_STATEMENT_FILE
+    python -m datatrails_scitt_samples.scripts.verify_receipt \
```

Comment:
There is a companion PR for datatrails_scitt_samples that needs to merge in order for this to work